src/redux/categories/categories.js

```diff
@@ -0,0 +1,20 @@
+const STATUS = 'STATUS';
```

Comment:
- [ ] Please follow the [docks pattern](https://github.com/erikras/ducks-modular-redux) for the value of each action type for reasons outlined in the comment above.

Reply:
This issue is well resolved! ✔️